extensions/mozilla-firefox/src/util/index.ts

```diff
@@ -1,5 +1,10 @@
 import fs from "fs";
 import path from "path";
+import { getPreferenceValues } from "@raycast/api";
+
+interface Preferences {
+  profileDirectorySuffix: string;
+}
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`x

Reply:
Ah thanks!